bin/donejs

```diff
@@ -46,6 +49,22 @@ utils.projectRoot().then(function (root) {
     catchAll: function (command) {
       console.error('Could not run `' + command + '`');
       program.help();
+    },
+    add: function(name, params) {
+      if(generators && generators[name]) {
+        return actions.generate(name, params);
+      }
+
+      var moduleName = 'donejs-' + name;
+
+      Q.ninvoke(npm, 'load', { loaded: false})
+        .then(utils.installIfMissing(path.join(root, 'node_modules'), moduleName))
+        .then(function() {
+          require(path.join(root, 'node_modules', moduleName));
```

Comment:
This is the issue that basically describes what it's trying to do: https://github.com/donejs/donejs/issues/202. Basically, it will install `donejs-cordova` from npm and run it.